src/field_theory/minpoly.lean

```diff
@@ -62,6 +65,18 @@ begin
   { exact aeval_zero _ }
 end
 
+lemma mem_range_of_degree_eq_one (hx : (minpoly A x).degree = 1) : x ∈ (algebra_map A B).range :=
+begin
+  have h : is_integral A x,
+  { by_contra,
```

Comment:
```suggestion
  { by_contra h,
```

Reply:
Otherwise the `h` on the line below is referring to a variable with an auto-generated name. This isn't very bad locally, but on a big scale it can cause headaches if in the future a tactic decides to generate a different name for some reason.